stripe/src/test/java/com/stripe/android/model/PaymentIntentTest.kt

```diff
@@ -16,40 +16,6 @@ import org.robolectric.RobolectricTestRunner
 @RunWith(RobolectricTestRunner::class)
 class PaymentIntentTest {
 
-    @Test
```

Comment:
you can remove all the companion object fields from this class

Reply:
oh yes